manifesto.md

```diff
@@ -5,56 +5,75 @@ title: MathML Manifesto
 
 Warning: this is a draft.
 
-1) Mathematical Language
+#### 1) Mathematical Language
 
-   The mathematical language is fundamental for educational and scientific
-   communication and must be handled by internet tools with the same
-   consideration as any other human language.
+Mathematical language is fundamental for scientific, technical, and educational
+communication.  It must be supported by internet tools with the same consideration as any
+other human language.
 
-2) Open Standards
+#### 2) Open Standards
 
-   In order to guarantee compatibility for data exchange, mathematical
-   rendering and authoring tools must rely on open formats and more
-   specifically on the MathML specification. When extension or improvements are
-   necessary, they must be conducted in a public standardization process.
+In order to guarantee compatibility for data exchange, mathematical rendering and
+authoring tools must rely on open formats and standards. When extension or improvements
+are necessary, they must be conducted in a public standardization process.
 
-3) Availability
+#### 3) Mathematical Formulae
 
-   MathML must be used on the internet to encode mathematical formulas. For
-   static formulas, it must be provided in the source code of web pages or
-   electronic mails in order to make it usable by the largest number of tools,
-   including web crawlers and browsers without support for script language such
-   as Javascript.
+Modern mathematical language relies on mathematical formulae, highly structured
+expressions composed of previously introduced identifiers that are presented in a
+conventionalized sesquidimensional notation.
```

Comment:
sesqui-something -is 1.5-something. So "sesquidimensional notation" is "1.5-dimensional notation"